RevenueCatUI/Templates/V2/Components/Button/BottomSheetView.swift

```diff
@@ -187,16 +191,18 @@ struct BottomSheetViewTestView: View {
             ]
         )))
     var body: some View {
-        ZStack {
-            Color.gray.opacity(0.2)
-
-            VStack {
-                Text("This view will have a sheet over it")
-                    .bottomSheet(sheet: $sheetViewModel,
-                                 safeAreaInsets: .init(top: 0, leading: 0, bottom: 0, trailing: 0))
+        GeometryReader { proxy in
+            ZStack {
+                Color.gray.opacity(0.2)
+
+                VStack {
+                    Text("This view will have a sheet over it")
+                        .bottomSheet(sheet: $sheetViewModel,
+                                     safeAreaInsets: proxy.safeAreaInsets)
+                }
             }
+            .edgesIgnoringSafeArea(.all)
         }
-        .edgesIgnoringSafeArea(.all)
```

Comment:
This just changes the preview to more closely resemble what we actually do in paywalls